app/views/layouts/_navbar.html.erb

```diff
@@ -18,7 +18,7 @@
           <a href="/listings">Listings</a>
         </div>
         <div class="navbar-item">
-          <a href="/external_resources">External Resources</a>
+          <a href="/community_resources">Community Resources</a>
```

Comment:
Change navbar to go to the public community_resources vs the admin external_resources